tensorflow/contrib/gdr/gdr_memory_manager.cc

```diff
@@ -640,8 +642,8 @@ void GdrMemoryManager::TensorFromTransportOptions(
     } else {
       checksum = GPUUtil::Checksum(*tensor);
     }
-    CHECK(checksum == remote_mr.checksum())
-        << "Checksum mismatch: " << checksum << "!=" << remote_mr.checksum();
+    CHECK(checksum == remote_mr.checksum()) << "Checksum mismatch: " << checksum
```

Comment:
Oh this is going to create a problem internally. Could you change that into a return error::InternalError?

Reply:
FYI: This only hits when turning VLOG>=2, and it is being used to debug memory corruption problems from hardware direct access. Like an assertion, and it’s easier to debug if we could keep the original stack instead of returning.

Thoughts?

Reply:
It's a policy that we have internally. You can LOG(ERROR) something and then return.

Reply:
sure, I will do it tomorrow morning.

Reply:
@drpngx  Will just re-committing the line back to the non-formatted version be sufficient to comply with the policy or do I need to force-push over the commit with that change?

Reply:
I guess the latter as they always squash the PR to a single commit :p

Reply:
Yes it doesn't matter how the commits are organized (I prefer a linear chain, just piling up so that we can match up the review history).

Reply:
done